client/components/registration/registration.js

```diff
@@ -299,7 +299,14 @@ export default class Registration extends React.Component {
   isPlanIdentityVerifier = () => {
     const {selected_plan, plans} = this.state;
     return (
-      selected_plan !== null && plans[selected_plan].verifies_identity === true
+      selected_plan !== null && plans[selected_plan].requires_payment === true
+    );
+  };
+
+  isPlanRequiresInvoice = () => {
+    const {selected_plan, plans} = this.state;
+    return (
+      selected_plan !== null && plans[selected_plan].requires_invoice === true
     );
   };
 
```

Comment:
maybe we can get rid of `isPlanIdentityVerifier` function?

Reply:
This is the only place where it is being used and I believe it can be removed without any problem.

Just need your confirmation @nemesisdesign 

https://github.com/openwisp/openwisp-wifi-login-pages/blob/e44bedc50280418b89ba18c4f39165ecebc54b87/client/components/registration/registration.js#L367-L381

Reply:
I think we can keep it and we can keep the name as is now (`isPlanIdentityVerifier`) with thefollowing comment:

if a payment is required, the plan is valid for identity verification.

It makes the code more readable.